### ISSUE TITLE
overlays not working?

### ISSUE THREAD

**Issue by arnonuem** [kartoza/docker-geoserver#23]
I am using docker-compose to combine docker-geoserver with some other containers.
In my docker-compose.yml i am pointing at my Dockerfile which basically just contains a 
`FROM kartoza/docker-geoserver`. Next to the Dockerfile i created a folder structure like: `resources/overlays/opt/geoserver/data_dir/workspaces/and_so_forth`. When using `docker-compose build` i expected to see that folders and files mirrored in my mapped data_dir volume. But nothing happens. When i exec /bin/bash into the geoserver container no extra folders can be found below `/opt/geoserver/data_dir`.
Do you have any suggestions why this fails?

The relevant part of my `docker-compose.yml` file looks like this:

```
  my-geoserver:
    build: my-geoserver/
    ports:
      - "8080:8080"
    volumes:
      - ./my-geoserver/data_dir:/opt/geoserver/data_dir
    links:
      - my-postgis
    restart: always
```

**Comment by timlinux** [comment]
Can you confirm this is not the same issue as #2 ?

**Comment by arnonuem** [comment]
Yes could be the same.

**Comment by NyakudyaA** [comment]
This has been resolved in the #2
